gwt-uibinder-processor/src/main/java/org/gwtproject/uibinder/processor/model/OwnerFieldClass.java

```diff
@@ -215,9 +215,7 @@ private ExecutableElement disambiguateSetters(String propertyName,
   private Multimap<String, ExecutableElement> findAllSetters(TypeMirror fieldType) {
     Multimap<String, ExecutableElement> allSetters = LinkedHashMultimap.create();
 
-    TypeElement fieldElement = AptUtil.asTypeElement(fieldType);
-    for (ExecutableElement method : ElementFilter
-        .methodsIn(fieldElement.getEnclosedElements())) {
+    for (ExecutableElement method : AptUtil.getInheritableMethods(fieldType)) {
```

Comment:
apparently I had already implemented a "getInheritableMethods" for UiHandler work later.   The reason we were seeing ambiguous setters is that it was just recursively grabbing all setters without identifying overridden ones.